src/main/scala/com/couchbase/spark/sql/N1QLRelation.scala

```diff
@@ -72,8 +72,17 @@ class N1QLRelation(bucket: String, userSchema: Option[StructType], parameters: M
       .map(_.value.toString)
     val dataset = sqlContext.sparkSession.createDataset(rdd)(Encoders.STRING)
 
+    val timestampFormat = if (parameters.get("timestampFormat").isDefined) {
+      parameters("timestampFormat")
+    } else {
+      // note: this is the default in Spark 2.3 FastDateParser;
+      //  assume couchbase should default to the same
+      "yyyy-MM-dd'T'HH:mm:ss.SSSXXX"
```

Comment:
This is taken from:
https://github.com/apache/spark/blob/branch-2.3/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/json/JSONOptions.scala#L84